asv.conf.json

```diff
@@ -10,10 +12,43 @@
     // list indicates to just test against the default (latest)
     // version.
     "matrix": {
-        "numpy": ["1.14"],
-        "numba": ["0.39"]
+        "numpy": [],
```

Comment:
Perhaps we should update the versions to something more recent, for example NumPy 1.16 and numba 0.46, but keep the versions anyway. This way we can control for performance changes when the dependencies upgrade.